### ISSUE TITLE
some plugins fail when no database is configured

### ISSUE THREAD

**Issue by anarcat** [embolalia/willie#668]
the weather plugin gives an exception out of the box:

```
02:49:54 <anarcat> .weather
02:49:54 <willcat> OperationalError: near "<": syntax error (file "/home/anarcat/src/willie/willie/db.py", line 756, in contains)
```

backtrace:

```
/home/anarcat/src/willie/willie/tools.py:282: UserWarning: containswill be removed in Willie 5.0. Please see http://willie.dftba.net/willie_5.html for more info.
  warnings.warn(thing + 'will be removed in Willie 5.0. Please see '
Traceback (most recent call last):
  File "/home/anarcat/src/willie/willie/bot.py", line 741, in call
    exit_code = func(willie, trigger)
  File "/home/anarcat/src/willie/willie/modules/weather.py", line 135, in weather
    if bot.db and trigger.nick in bot.db.preferences:
  File "/home/anarcat/src/willie/willie/db.py", line 765, in __contains__
    return self.contains(item)
  File "/home/anarcat/src/willie/willie/tools.py", line 289, in new
    return old(*args, **kwargs)
  File "/home/anarcat/src/willie/willie/db.py", line 756, in contains
    cur.execute('SELECT * FROM ' + self.name + ' WHERE ' + where, [row])
OperationalError: near "<": syntax error
```

1. the above query should use placeholders (ie. `cur.execute('SELECT * FROM %s WHERE %s', (self.name, where % [row]))` - guessing here)
2. second, it shouldn't give an exception, even if the database is not configured (which is what I suspect the underlying problem is here)

this applies to all database plugins, and only on git head (Willie v. 4.5.1-git at commit: cd36f6051c963e24e12f6580570c35fe3ebeeb99). 4.5.1 of course works fine.

**Comment by embolalia** [comment]
This was fixed by d78ceb6, and then obviated by 5.0, so it probably doesn't need to be open anymore.

**Comment by anarcat** [comment]
i confirm this is fixed in 5.0, thanks.
